ouroboros-consensus-cardano-tools/src/Cardano/Tools/DBAnalyser/Analysis.hs

```diff
@@ -782,10 +782,11 @@ reproMempoolForge numBlks env = do
             let slot = blockSlot blk
             (ticked, durTick) <- timed $ IOLike.evaluate $
               applyChainTick lCfg slot (ledgerState $ LedgerDB.current ldb)
+            vh <- lbsValueHandle bstore
             ((), durSnap) <- timed $ do
-              snap <- Mempool.getSnapshotFor mempool (castPoint $ getTip $ LedgerDB.current ldb) slot ticked
-
-              pure $ length . Mempool.snapshotTxs <$> snap `seq` Mempool.snapshotState <$> snap `seq` ()
+              snap <- Mempool.getSnapshotFor mempool slot ticked ldb vh
+              pure $ length (Mempool.snapshotTxs snap) `seq` Mempool.snapshotState snap `seq` ()
```

Comment:
This was not introduced in this PR but I wonder if we should comment on why we need these multiple `seq`s.

Reply:
#4493